From 1d7c4072a5be6f3bf57f8296fe2e2545a18c2fe2 Mon Sep 17 00:00:00 2001
From: "Brett T. Warden" <brett.t.warden@intel.com>
Date: Fri, 30 Oct 2015 16:13:35 -0700
Subject: [PATCH 221/639] dwc3: fix super speed enumeration issue for CGM

Apply the CHT fix from adadfa2074d8b161f9b293aa4a27b12a0713ea53 to
derivative CGM.

Signed-off-by: Brett T. Warden <brett.t.warden@intel.com>
---
 drivers/usb/dwc3/dwc3-pci.c | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/drivers/usb/dwc3/dwc3-pci.c b/drivers/usb/dwc3/dwc3-pci.c
index 243717e..f788b36 100644
--- a/drivers/usb/dwc3/dwc3-pci.c
+++ b/drivers/usb/dwc3/dwc3-pci.c
@@ -280,7 +280,10 @@ static const struct pci_device_id dwc3_pci_id_table[] = {
 		PCI_DEVICE(PCI_VENDOR_ID_INTEL, PCI_DEVICE_ID_INTEL_CHT),
 		.driver_data	= (kernel_ulong_t)&dwc3_intel_cht,
 	},
-	{ PCI_DEVICE(PCI_VENDOR_ID_INTEL, PCI_DEVICE_ID_INTEL_CGM), },
+	{
+		PCI_DEVICE(PCI_VENDOR_ID_INTEL, PCI_DEVICE_ID_INTEL_CGM),
+		.driver_data	= (kernel_ulong_t)&dwc3_intel_cht,
+	},
 	{  }	/* Terminating Entry */
 };
 MODULE_DEVICE_TABLE(pci, dwc3_pci_id_table);
-- 
2.10.1

